client/js/screens/SingleSelfGuided/SingleSelfGuided.js

```diff
@@ -0,0 +1,168 @@
+import React, {Component} from 'react';
+import {View, Image, Text, ScrollView, TouchableOpacity} from 'react-native';
+import ButtonDefault from '../../components/ButtonDefault';
+import styles from './styles';
+import SafeAreaView from 'react-native-safe-area-view';
+import MapView from 'react-native-maps';
+import {Marker} from 'react-native-maps';
+
+const SingleSelfGuided = ({navigation, tour}) => {
+  return (
+    <>
```

Comment:
Please remove your Fragment wrapper. No need as the SafeAreaView is the only child it has.

Reply:
Removed